.env

```diff
@@ -16,6 +16,7 @@ IDAM_KEY_CCD_GATEWAY=AAAAAAAAAAAAAAAC
 IDAM_KEY_CCD_ADMIN=AAAAAAAAAAAAAAAD
```

Comment:
for my information, will ccd-docker in the future be updated to start up XUI components as well?

Reply:
XUI docker support does not exists yet. If you dont mind I would like to start XUI service from ccd-docker compose so that other teams can benefit from it, but I'm not sure if this valid solution. For now I decided to make minimal changes in ccd-docker (whitelisting in ccd services + s2s client registration) and intention is to starts XUI service from our compose file (fpla)